README.md

```diff
@@ -49,14 +45,22 @@ You can activate the following optional profiles (`build-assembly` is active by
 - `-P '!build-assembly'`: to not build the assembly, e.g. only build the library part of Antenna.
 
 ### Configure Antenna
+<<<<<<< HEAD
 Antenna can be used as a Maven plugin, with  Gradle or standalone executable.
 As a maven plugin, Antenna's behaviour is configured by adding a `<plugin>` to your project's `pom.xml` file and adding settings to the `<configuration>` section.
 Similarly, in Gradle, the same Maven files must be given and the `build.gradle` file needs to include the Antenna configuration.
 As a standalone executable, Antenna is configured as an executable jar in the command line.
 Find out how to configure Antenna by reading: [How to configure Antenna](antenna-documentation/src/site/markdown/how-to-configure.md.vm).
+=======
+Antenna can be used as a Maven plugin or standalone executable. As a 
+maven plugin, Antenna's behaviour is configured by adding a `<plugin>` to your
+project's `pom.xml` file and adding settings to the `<configuration>`
+section. As a standalone executable is configured as an executable jar in the command line. 
+Find out how to configure Antenna by reading [How to configure Antenna](antenna-documentation/src/site/markdown/how-to-configure.md). 
+>>>>>>> fix(doc): gradle plugin site now working
```

Comment:
@neubs-bsi This is a relict from rebasing. Please change.

Reply:
Thanks, I removed it.